From 2a1307f2b116e136727a338137d8f21d3bda57e8 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Thu, 27 Jan 2022 14:18:03 +0200
Subject: [PATCH] [SW-0] uart: fix preboot.h include

preboot.h is not located in the local folder and thus is part of the
include list provided by the build system.
Use <> for including preboot.h.

Change-Id: Iffc08accca013a376dd40dc3eb1abe13e45f37b6
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/lib/common/uart.c b/lib/common/uart.c
index 9f59d49..65c303b 100644
--- a/lib/common/uart.c
+++ b/lib/common/uart.c
@@ -8,7 +8,7 @@
 
 #include <psoc_uart_0_regs.h>
 
-#include "preboot.h"
+#include <preboot.h>
 #include "uart.h"
 #include "pll.h"
 
